api/v1alpha1/virtualmachine_conversion.go

```diff
@@ -1138,6 +1138,20 @@ func Convert_v1alpha3_VirtualMachine_To_v1alpha1_VirtualMachine(in *vmopv1.Virtu
 	// necessary for later versions that don't support configMap based bootstrap.
 	delete(out.Annotations, vmopv1.V1alpha1ConfigMapTransportAnnotation)
 
+	// Handle the renaming of pause annotation on down convert.
+	if val, ok := in.Annotations[vmopv1.PauseAnnotation]; ok {
+		annotations := out.GetAnnotations()
+		if annotations == nil {
+			annotations = make(map[string]string)
```

Comment:
Aren't these missing a `SetAnnotation()` for this case?

Reply:
Thanks for spotting this.  Fixed.